framework/src/base/Assembly.C

```diff
@@ -939,7 +943,12 @@ Assembly::prepare()
     unsigned int vi = ivar.number();
     unsigned int vj = jvar.number();
 
-    jacobianBlock(vi, vj).resize(ivar.dofIndices().size(), jvar.dofIndices().size());
+    AllLocalDofIndicesThread aldit(_sys.system(), {jvar.name()});
```

Comment:
This is not at all where you want to do this... you realize that it's going to do this on _every_ element, right?  You want to do this _once_ up front and store away the results somewhere.

That's not why it's failing (I'm still looking at that)... but this has to change anyway...